modules/developer-cli-odo-switching-between-ephemeral-and-persistent-storage.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// *cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/working-with-storage.adoc
+
+[id="switching-between-ephemeral-and-persistent-storage_{context}"]
+= Switching between ephemeral and persistent storage
+
+You can switch between ephemeral and persistent storage in your project by using the `odo preference` command. `odo preference` modifies the global preference in your cluster.
+
+When persistent storage is enabled, the cluster stores the information between the restarts. 
+
+When ephemeral storage is enabled, the cluster does not store the information between the restarts.
+
+Ephemeral storage is enabled by default. 
+
+.Procedure
+
+. See the preference currently set in your project:
++
+[source,terminal]
+----
+odo preference view
```

Comment:
I think this needs a command prompt $ before the command itself. Same for the other commands in this module.

Reply:
@ahardin-rh thanks! Added the prompt.